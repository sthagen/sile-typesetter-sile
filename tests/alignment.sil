\begin[papersize=a6]{document}
\nofolios
\use[module=packages.lorem]
\font[family=Libertinus Serif]
\set[parameter=document.parskip,value=10pt]

\raggedleft{\em{Ragged left} \lorem[words=40]}

\raggedright{\em{Ragged right} {\lorem[words=40]}}

\center{\em{Centered} \lorem[words=40]}

\justified{\em{Justified} \lorem[words=40]}

\raggedleft{\em{Right aligned} \lorem[words=4]}

\end{document}
