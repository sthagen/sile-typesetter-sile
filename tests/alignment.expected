Set paper size 	297.6377985	419.5275636
Begin page
Mx 	33.9059
My 	27.9564
Set font 	Libertinus Serif;10;400;Italic;normal;;;LTR
Mx 	5.8700
Mx 	4.5700
Mx 	5.0000
Mx 	5.0000
Mx 	4.5400
Mx 	5.0600
T	51 a=5.8700 66 a=4.5700 72 a=5.0000 72 a=5.0000 70 a=4.4700 69 a=5.0600	(Ragged)
Mx 	66.4459
T	77 70 2385 w=13.0700	(left)
Mx 	82.0159
Set font 	Libertinus Serif;10;400;;normal;;;LTR
Mx 	2.6400
Mx 	5.0400
Mx 	3.6400
Mx 	4.4700
Mx 	7.9000
T	77 a=2.6400 80 a=5.0400 83 a=3.7200 70 a=4.4700 78 a=7.9000	(lorem)
Mx 	108.2059
T	74 81 84 86 78 w=25.0100	(ipsum)
Mx 	135.7159
T	69 80 77 80 83 w=21.5000	(dolor)
Mx 	159.7159
T	84 74 85 w=9.7700	(sit)
Mx 	171.9859
T	66 78 70 85 w=20.1000	(amet)
Mx 	194.5859
T	68 80 79 84 70 85 70 85 86 83 w=42.9300	(consetetur)
Mx 	240.0159
T	84 66 69 74 81 84 68 74 79 72 w=42.7400	(sadipscing)
Mx 	14.8819
My 	39.9564
T	70 77 74 85 83 w=16.7000	(elitr)
Mx 	33.8353
Mx 	3.9000
Mx 	4.5400
Mx 	5.0600
T	84 a=3.9000 70 a=4.4700 69 a=5.0600	(sed)
Mx 	49.5887
T	69 74 66 78 w=20.2400	(diam)
Mx 	72.0821
T	79 80 79 86 78 90 w=34.2400	(nonumy)
Mx 	108.5755
Mx 	4.4700
Mx 	2.7100
Mx 	3.7200
Mx 	7.9000
Mx 	5.1100
Mx 	5.0600
T	70 a=4.4700 74 a=2.7100 83 a=3.7200 78 a=7.9000 80 a=5.0400 69 a=5.0600	(eirmod)
Mx 	139.7989
Mx 	3.1600
Mx 	4.4700
Mx 	7.9000
Mx 	5.2600
Mx 	5.0400
Mx 	3.7200
T	85 a=3.1600 70 a=4.4700 78 a=7.9000 81 a=5.1900 80 a=5.0400 83 a=3.7200	(tempor)
Mx 	171.6023
T	74 79 87 74 69 86 79 85 w=34.7600	(invidunt)
Mx 	208.6157
T	86 85 w=8.4700	(ut)
Mx 	219.3391
Mx 	2.6400
Mx 	4.5700
Mx 	5.0300
Mx 	5.0400
Mx 	3.6400
Mx 	4.4700
T	77 a=2.6400 66 a=4.5700 67 a=4.9300 80 a=5.0400 83 a=3.7200 70 a=4.4700	(labore)
Mx 	246.9825
T	70 85 w=7.6300	(et)
Mx 	256.8659
Mx 	5.0600
Mx 	5.0400
Mx 	2.6400
Mx 	5.0400
Mx 	3.6400
Mx 	4.4700
T	69 a=5.0600 80 a=5.0400 77 a=2.6400 80 a=5.0400 83 a=3.7200 70 a=4.4700	(dolore)
Mx 	14.8819
My 	51.9564
T	78 66 72 79 66 w=27.4600	(magna)
Mx 	44.8368
T	66 77 74 82 86 90 66 78 w=37.8800	(aliquyam)
Mx 	85.2117
T	70 83 66 85 w=15.9200	(erat)
Mx 	103.6266
Mx 	3.9000
Mx 	4.5400
Mx 	5.0600
T	84 a=3.9000 70 a=4.4700 69 a=5.0600	(sed)
Mx 	119.6215
T	69 74 66 78 w=20.2400	(diam)
Mx 	142.3564
Mx 	4.8900
Mx 	5.0400
Mx 	2.6400
Mx 	5.3100
Mx 	5.1900
Mx 	3.1600
Mx 	5.3100
Mx 	4.5700
T	87 a=4.9700 80 a=5.0400 77 a=2.6400 86 a=5.3100 81 a=5.1900 85 a=3.1600 86 a=5.3100 66 a=4.5700	(voluptua)
Mx 	180.9614
T	66 85 w=7.7300	(at)
Mx 	191.1863
Mx 	4.8900
Mx 	4.4700
Mx 	3.6400
Mx 	5.0400
T	87 a=4.9700 70 a=4.4700 83 a=3.7200 80 a=5.0400	(vero)
Mx 	211.7212
Mx 	4.5400
Mx 	5.0400
Mx 	3.9000
T	70 a=4.4700 80 a=5.0400 84 a=3.9000	(eos)
Mx 	227.6961
T	70 85 w=7.6300	(et)
Mx 	237.8210
T	66 68 68 86 84 66 78 w=34.8100	(accusam)
Mx 	275.1259
T	70 85 w=7.6300	(et)
Mx 	59.2459
My 	63.9564
T	75 86 84 85 80 w=20.1300	(justo)
Mx 	81.8759
T	69 86 80 w=15.4100	(duo)
Mx 	99.7859
Mx 	5.0600
Mx 	5.0400
Mx 	2.6400
Mx 	5.0400
Mx 	3.6400
Mx 	4.4700
Mx 	3.9000
T	69 a=5.0600 80 a=5.0400 77 a=2.6400 80 a=5.0400 83 a=3.7200 70 a=4.4700 84 a=3.9000	(dolores)
Mx 	132.0759
T	70 85 w=7.6300	(et)
Mx 	142.2059
T	70 66 w=9.0400	(ea)
Mx 	153.7459
Mx 	3.6400
Mx 	4.4700
Mx 	4.9300
Mx 	5.3100
Mx 	7.9000
T	83 a=3.7200 70 a=4.4700 67 a=4.9300 86 a=5.3100 78 a=7.9000	(rebum)
Mx 	182.4959
T	84 85 70 85 w=14.6900	(stet)
Mx 	199.6859
T	68 77 74 85 66 w=17.3600	(clita)
Mx 	219.5459
T	76 66 84 69 w=18.6500	(kasd)
Mx 	240.6959
Mx 	5.0000
Mx 	5.3100
Mx 	5.0300
Mx 	4.4700
Mx 	3.7200
Mx 	5.0000
Mx 	3.6400
Mx 	4.4700
Mx 	5.4200
T	72 a=5.0000 86 a=5.3100 67 a=4.9300 70 a=4.4700 83 a=3.7200 72 a=5.0000 83 a=3.7200 70 a=4.4700 79 a=5.4200	(gubergren)
Mx 	26.8819
My 	95.9564
Set font 	Libertinus Serif;10;400;Italic;normal;;;LTR
Mx 	5.8700
Mx 	4.5700
Mx 	5.0000
Mx 	5.0000
Mx 	4.5400
Mx 	5.0600
T	51 a=5.8700 66 a=4.5700 72 a=5.0000 72 a=5.0000 70 a=4.4700 69 a=5.0600	(Ragged)
Mx 	59.4219
T	83 74 72 73 85 w=19.9700	(right)
Mx 	81.8919
Set font 	Libertinus Serif;10;400;;normal;;;LTR
Mx 	2.6400
Mx 	5.0400
Mx 	3.6400
Mx 	4.4700
Mx 	7.9000
T	77 a=2.6400 80 a=5.0400 83 a=3.7200 70 a=4.4700 78 a=7.9000	(lorem)
Mx 	108.0819
T	74 81 84 86 78 w=25.0100	(ipsum)
Mx 	135.5919
T	69 80 77 80 83 w=21.5000	(dolor)
Mx 	159.5919
T	84 74 85 w=9.7700	(sit)
Mx 	171.8619
T	66 78 70 85 w=20.1000	(amet)
Mx 	194.4619
T	68 80 79 84 70 85 70 85 86 83 w=42.9300	(consetetur)
Mx 	239.8919
T	84 66 69 74 81 84 68 74 79 72 w=42.7400	(sadipscing)
Mx 	14.8819
My 	107.9564
T	70 77 74 85 83 w=16.7000	(elitr)
Mx 	33.8353
Mx 	3.9000
Mx 	4.5400
Mx 	5.0600
T	84 a=3.9000 70 a=4.4700 69 a=5.0600	(sed)
Mx 	49.5887
T	69 74 66 78 w=20.2400	(diam)
Mx 	72.0821
T	79 80 79 86 78 90 w=34.2400	(nonumy)
Mx 	108.5755
Mx 	4.4700
Mx 	2.7100
Mx 	3.7200
Mx 	7.9000
Mx 	5.1100
Mx 	5.0600
T	70 a=4.4700 74 a=2.7100 83 a=3.7200 78 a=7.9000 80 a=5.0400 69 a=5.0600	(eirmod)
Mx 	139.7989
Mx 	3.1600
Mx 	4.4700
Mx 	7.9000
Mx 	5.2600
Mx 	5.0400
Mx 	3.7200
T	85 a=3.1600 70 a=4.4700 78 a=7.9000 81 a=5.1900 80 a=5.0400 83 a=3.7200	(tempor)
Mx 	171.6023
T	74 79 87 74 69 86 79 85 w=34.7600	(invidunt)
Mx 	208.6157
T	86 85 w=8.4700	(ut)
Mx 	219.3391
Mx 	2.6400
Mx 	4.5700
Mx 	5.0300
Mx 	5.0400
Mx 	3.6400
Mx 	4.4700
T	77 a=2.6400 66 a=4.5700 67 a=4.9300 80 a=5.0400 83 a=3.7200 70 a=4.4700	(labore)
Mx 	246.9825
T	70 85 w=7.6300	(et)
Mx 	256.8659
Mx 	5.0600
Mx 	5.0400
Mx 	2.6400
Mx 	5.0400
Mx 	3.6400
Mx 	4.4700
T	69 a=5.0600 80 a=5.0400 77 a=2.6400 80 a=5.0400 83 a=3.7200 70 a=4.4700	(dolore)
Mx 	14.8819
My 	119.9564
T	78 66 72 79 66 w=27.4600	(magna)
Mx 	44.8368
T	66 77 74 82 86 90 66 78 w=37.8800	(aliquyam)
Mx 	85.2117
T	70 83 66 85 w=15.9200	(erat)
Mx 	103.6266
Mx 	3.9000
Mx 	4.5400
Mx 	5.0600
T	84 a=3.9000 70 a=4.4700 69 a=5.0600	(sed)
Mx 	119.6215
T	69 74 66 78 w=20.2400	(diam)
Mx 	142.3564
Mx 	4.8900
Mx 	5.0400
Mx 	2.6400
Mx 	5.3100
Mx 	5.1900
Mx 	3.1600
Mx 	5.3100
Mx 	4.5700
T	87 a=4.9700 80 a=5.0400 77 a=2.6400 86 a=5.3100 81 a=5.1900 85 a=3.1600 86 a=5.3100 66 a=4.5700	(voluptua)
Mx 	180.9614
T	66 85 w=7.7300	(at)
Mx 	191.1863
Mx 	4.8900
Mx 	4.4700
Mx 	3.6400
Mx 	5.0400
T	87 a=4.9700 70 a=4.4700 83 a=3.7200 80 a=5.0400	(vero)
Mx 	211.7212
Mx 	4.5400
Mx 	5.0400
Mx 	3.9000
T	70 a=4.4700 80 a=5.0400 84 a=3.9000	(eos)
Mx 	227.6961
T	70 85 w=7.6300	(et)
Mx 	237.8210
T	66 68 68 86 84 66 78 w=34.8100	(accusam)
Mx 	275.1259
T	70 85 w=7.6300	(et)
Mx 	14.8819
My 	131.9564
T	75 86 84 85 80 w=20.1300	(justo)
Mx 	37.5119
T	69 86 80 w=15.4100	(duo)
Mx 	55.4219
Mx 	5.0600
Mx 	5.0400
Mx 	2.6400
Mx 	5.0400
Mx 	3.6400
Mx 	4.4700
Mx 	3.9000
T	69 a=5.0600 80 a=5.0400 77 a=2.6400 80 a=5.0400 83 a=3.7200 70 a=4.4700 84 a=3.9000	(dolores)
Mx 	87.7119
T	70 85 w=7.6300	(et)
Mx 	97.8419
T	70 66 w=9.0400	(ea)
Mx 	109.3819
Mx 	3.6400
Mx 	4.4700
Mx 	4.9300
Mx 	5.3100
Mx 	7.9000
T	83 a=3.7200 70 a=4.4700 67 a=4.9300 86 a=5.3100 78 a=7.9000	(rebum)
Mx 	138.1319
T	84 85 70 85 w=14.6900	(stet)
Mx 	155.3219
T	68 77 74 85 66 w=17.3600	(clita)
Mx 	175.1819
T	76 66 84 69 w=18.6500	(kasd)
Mx 	196.3319
Mx 	5.0000
Mx 	5.3100
Mx 	5.0300
Mx 	4.4700
Mx 	3.7200
Mx 	5.0000
Mx 	3.6400
Mx 	4.4700
Mx 	5.4200
T	72 a=5.0000 86 a=5.3100 67 a=4.9300 70 a=4.4700 83 a=3.7200 72 a=5.0000 83 a=3.7200 70 a=4.4700 79 a=5.4200	(gubergren)
Mx 	18.9889
My 	163.9564
Set font 	Libertinus Serif;10;400;Italic;normal;;;LTR
Mx 	6.4600
Mx 	4.4700
Mx 	5.4200
Mx 	3.1600
Mx 	4.4700
Mx 	3.6400
Mx 	4.5400
Mx 	5.0600
T	36 a=6.4600 70 a=4.4700 79 a=5.4200 85 a=3.1600 70 a=4.4700 83 a=3.7200 70 a=4.4700 69 a=5.0600	(Centered)
Mx 	58.7089
Set font 	Libertinus Serif;10;400;;normal;;;LTR
Mx 	2.6400
Mx 	5.0400
Mx 	3.6400
Mx 	4.4700
Mx 	7.9000
T	77 a=2.6400 80 a=5.0400 83 a=3.7200 70 a=4.4700 78 a=7.9000	(lorem)
Mx 	84.8989
T	74 81 84 86 78 w=25.0100	(ipsum)
Mx 	112.4089
T	69 80 77 80 83 w=21.5000	(dolor)
Mx 	136.4089
T	84 74 85 w=9.7700	(sit)
Mx 	148.6789
T	66 78 70 85 w=20.1000	(amet)
Mx 	171.2789
T	68 80 79 84 70 85 70 85 86 83 w=42.9300	(consetetur)
Mx 	216.7089
T	84 66 69 74 81 84 68 74 79 72 w=42.7400	(sadipscing)
Mx 	261.9489
T	70 77 74 85 83 w=16.7000	(elitr)
Mx 	23.2489
My 	175.9564
Mx 	3.9000
Mx 	4.5400
Mx 	5.0600
T	84 a=3.9000 70 a=4.4700 69 a=5.0600	(sed)
Mx 	39.2489
T	69 74 66 78 w=20.2400	(diam)
Mx 	61.9889
T	79 80 79 86 78 90 w=34.2400	(nonumy)
Mx 	98.7289
Mx 	4.4700
Mx 	2.7100
Mx 	3.7200
Mx 	7.9000
Mx 	5.1100
Mx 	5.0600
T	70 a=4.4700 74 a=2.7100 83 a=3.7200 78 a=7.9000 80 a=5.0400 69 a=5.0600	(eirmod)
Mx 	130.1989
Mx 	3.1600
Mx 	4.4700
Mx 	7.9000
Mx 	5.2600
Mx 	5.0400
Mx 	3.7200
T	85 a=3.1600 70 a=4.4700 78 a=7.9000 81 a=5.1900 80 a=5.0400 83 a=3.7200	(tempor)
Mx 	162.2489
T	74 79 87 74 69 86 79 85 w=34.7600	(invidunt)
Mx 	199.5089
T	86 85 w=8.4700	(ut)
Mx 	210.4789
Mx 	2.6400
Mx 	4.5700
Mx 	5.0300
Mx 	5.0400
Mx 	3.6400
Mx 	4.4700
T	77 a=2.6400 66 a=4.5700 67 a=4.9300 80 a=5.0400 83 a=3.7200 70 a=4.4700	(labore)
Mx 	238.3689
T	70 85 w=7.6300	(et)
Mx 	248.4989
Mx 	5.0600
Mx 	5.0400
Mx 	2.6400
Mx 	5.0400
Mx 	3.6400
Mx 	4.4700
T	69 a=5.0600 80 a=5.0400 77 a=2.6400 80 a=5.0400 83 a=3.7200 70 a=4.4700	(dolore)
Mx 	14.8819
My 	187.9564
T	78 66 72 79 66 w=27.4600	(magna)
Mx 	44.8368
T	66 77 74 82 86 90 66 78 w=37.8800	(aliquyam)
Mx 	85.2117
T	70 83 66 85 w=15.9200	(erat)
Mx 	103.6266
Mx 	3.9000
Mx 	4.5400
Mx 	5.0600
T	84 a=3.9000 70 a=4.4700 69 a=5.0600	(sed)
Mx 	119.6215
T	69 74 66 78 w=20.2400	(diam)
Mx 	142.3564
Mx 	4.8900
Mx 	5.0400
Mx 	2.6400
Mx 	5.3100
Mx 	5.1900
Mx 	3.1600
Mx 	5.3100
Mx 	4.5700
T	87 a=4.9700 80 a=5.0400 77 a=2.6400 86 a=5.3100 81 a=5.1900 85 a=3.1600 86 a=5.3100 66 a=4.5700	(voluptua)
Mx 	180.9614
T	66 85 w=7.7300	(at)
Mx 	191.1863
Mx 	4.8900
Mx 	4.4700
Mx 	3.6400
Mx 	5.0400
T	87 a=4.9700 70 a=4.4700 83 a=3.7200 80 a=5.0400	(vero)
Mx 	211.7212
Mx 	4.5400
Mx 	5.0400
Mx 	3.9000
T	70 a=4.4700 80 a=5.0400 84 a=3.9000	(eos)
Mx 	227.6961
T	70 85 w=7.6300	(et)
Mx 	237.8210
T	66 68 68 86 84 66 78 w=34.8100	(accusam)
Mx 	275.1259
T	70 85 w=7.6300	(et)
Mx 	37.0639
My 	199.9564
T	75 86 84 85 80 w=20.1300	(justo)
Mx 	59.6939
T	69 86 80 w=15.4100	(duo)
Mx 	77.6039
Mx 	5.0600
Mx 	5.0400
Mx 	2.6400
Mx 	5.0400
Mx 	3.6400
Mx 	4.4700
Mx 	3.9000
T	69 a=5.0600 80 a=5.0400 77 a=2.6400 80 a=5.0400 83 a=3.7200 70 a=4.4700 84 a=3.9000	(dolores)
Mx 	109.8939
T	70 85 w=7.6300	(et)
Mx 	120.0239
T	70 66 w=9.0400	(ea)
Mx 	131.5639
Mx 	3.6400
Mx 	4.4700
Mx 	4.9300
Mx 	5.3100
Mx 	7.9000
T	83 a=3.7200 70 a=4.4700 67 a=4.9300 86 a=5.3100 78 a=7.9000	(rebum)
Mx 	160.3139
T	84 85 70 85 w=14.6900	(stet)
Mx 	177.5039
T	68 77 74 85 66 w=17.3600	(clita)
Mx 	197.3639
T	76 66 84 69 w=18.6500	(kasd)
Mx 	218.5139
Mx 	5.0000
Mx 	5.3100
Mx 	5.0300
Mx 	4.4700
Mx 	3.7200
Mx 	5.0000
Mx 	3.6400
Mx 	4.4700
Mx 	5.4200
T	72 a=5.0000 86 a=5.3100 67 a=4.9300 70 a=4.4700 83 a=3.7200 72 a=5.0000 83 a=3.7200 70 a=4.4700 79 a=5.4200	(gubergren)
Mx 	26.8819
My 	231.9564
Set font 	Libertinus Serif;10;400;Italic;normal;;;LTR
Mx 	3.2200
Mx 	5.3100
Mx 	3.9000
Mx 	3.1600
Mx 	2.7100
Mx 	5.6000
Mx 	4.5400
Mx 	5.0600
T	43 a=3.2200 86 a=5.3100 84 a=3.9000 85 a=3.1600 74 a=2.7100 2361 a=5.6000 70 a=4.4700 69 a=5.0600	(Justified)
Mx 	62.8736
Set font 	Libertinus Serif;10;400;;normal;;;LTR
Mx 	2.6400
Mx 	5.0400
Mx 	3.6400
Mx 	4.4700
Mx 	7.9000
T	77 a=2.6400 80 a=5.0400 83 a=3.7200 70 a=4.4700 78 a=7.9000	(lorem)
Mx 	89.0554
T	74 81 84 86 78 w=25.0100	(ipsum)
Mx 	116.5571
T	69 80 77 80 83 w=21.5000	(dolor)
Mx 	140.5489
T	84 74 85 w=9.7700	(sit)
Mx 	152.8107
T	66 78 70 85 w=20.1000	(amet)
Mx 	175.4024
T	68 80 79 84 70 85 70 85 86 83 w=42.9300	(consetetur)
Mx 	220.8242
T	84 66 69 74 81 84 68 74 79 72 w=42.7400	(sadipscing)
Mx 	266.0559
T	70 77 74 85 83 w=16.7000	(elitr)
Mx 	14.8819
My 	243.9564
Mx 	3.9000
Mx 	4.5400
Mx 	5.0600
T	84 a=3.9000 70 a=4.4700 69 a=5.0600	(sed)
Mx 	30.7203
T	69 74 66 78 w=20.2400	(diam)
Mx 	53.2987
T	79 80 79 86 78 90 w=34.2400	(nonumy)
Mx 	89.8771
Mx 	4.4700
Mx 	2.7100
Mx 	3.7200
Mx 	7.9000
Mx 	5.1100
Mx 	5.0600
T	70 a=4.4700 74 a=2.7100 83 a=3.7200 78 a=7.9000 80 a=5.0400 69 a=5.0600	(eirmod)
Mx 	121.1855
Mx 	3.1600
Mx 	4.4700
Mx 	7.9000
Mx 	5.2600
Mx 	5.0400
Mx 	3.7200
T	85 a=3.1600 70 a=4.4700 78 a=7.9000 81 a=5.1900 80 a=5.0400 83 a=3.7200	(tempor)
Mx 	153.0739
T	74 79 87 74 69 86 79 85 w=34.7600	(invidunt)
Mx 	190.1723
T	86 85 w=8.4700	(ut)
Mx 	200.9807
Mx 	2.6400
Mx 	4.5700
Mx 	5.0300
Mx 	5.0400
Mx 	3.6400
Mx 	4.4700
T	77 a=2.6400 66 a=4.5700 67 a=4.9300 80 a=5.0400 83 a=3.7200 70 a=4.4700	(labore)
Mx 	228.7091
T	70 85 w=7.6300	(et)
Mx 	238.6775
Mx 	5.0600
Mx 	5.0400
Mx 	2.6400
Mx 	5.0400
Mx 	3.6400
Mx 	4.4700
T	69 a=5.0600 80 a=5.0400 77 a=2.6400 80 a=5.0400 83 a=3.7200 70 a=4.4700	(dolore)
Mx 	266.9059
T	78 66 w=12.4700	(ma)
Mx 	279.3759
T	14 w=3.3800	(-)
Mx 	14.8819
My 	255.9564
T	72 79 66 w=14.9900	(gna)
Mx 	33.5004
T	66 77 74 82 86 90 66 78 w=37.8800	(aliquyam)
Mx 	75.0090
T	70 83 66 85 w=15.9200	(erat)
Mx 	94.5575
Mx 	3.9000
Mx 	4.5400
Mx 	5.0600
T	84 a=3.9000 70 a=4.4700 69 a=5.0600	(sed)
Mx 	111.6861
T	69 74 66 78 w=20.2400	(diam)
Mx 	135.5546
Mx 	4.8900
Mx 	5.0400
Mx 	2.6400
Mx 	5.3100
Mx 	5.1900
Mx 	3.1600
Mx 	5.3100
Mx 	4.5700
T	87 a=4.9700 80 a=5.0400 77 a=2.6400 86 a=5.3100 81 a=5.1900 85 a=3.1600 86 a=5.3100 66 a=4.5700	(voluptua)
Mx 	175.2932
T	66 85 w=7.7300	(at)
Mx 	186.6517
Mx 	4.8900
Mx 	4.4700
Mx 	3.6400
Mx 	5.0400
T	87 a=4.9700 70 a=4.4700 83 a=3.7200 80 a=5.0400	(vero)
Mx 	208.3203
Mx 	4.5400
Mx 	5.0400
Mx 	3.9000
T	70 a=4.4700 80 a=5.0400 84 a=3.9000	(eos)
Mx 	225.4288
T	70 85 w=7.6300	(et)
Mx 	236.6874
T	66 68 68 86 84 66 78 w=34.8100	(accusam)
Mx 	275.1259
T	70 85 w=7.6300	(et)
Mx 	14.8819
My 	267.9564
T	75 86 84 85 80 w=20.1300	(justo)
Mx 	37.5174
T	69 86 80 w=15.4100	(duo)
Mx 	55.4330
Mx 	5.0600
Mx 	5.0400
Mx 	2.6400
Mx 	5.0400
Mx 	3.6400
Mx 	4.4700
Mx 	3.9000
T	69 a=5.0600 80 a=5.0400 77 a=2.6400 80 a=5.0400 83 a=3.7200 70 a=4.4700 84 a=3.9000	(dolores)
Mx 	87.7285
T	70 85 w=7.6300	(et)
Mx 	97.8640
T	70 66 w=9.0400	(ea)
Mx 	109.4096
Mx 	3.6400
Mx 	4.4700
Mx 	4.9300
Mx 	5.3100
Mx 	7.9000
T	83 a=3.7200 70 a=4.4700 67 a=4.9300 86 a=5.3100 78 a=7.9000	(rebum)
Mx 	138.1651
T	84 85 70 85 w=14.6900	(stet)
Mx 	155.3607
T	68 77 74 85 66 w=17.3600	(clita)
Mx 	175.2262
T	76 66 84 69 w=18.6500	(kasd)
Mx 	196.3817
Mx 	5.0000
Mx 	5.3100
Mx 	5.0300
Mx 	4.4700
Mx 	3.7200
Mx 	5.0000
Mx 	3.6400
Mx 	4.4700
Mx 	5.4200
T	72 a=5.0000 86 a=5.3100 67 a=4.9300 70 a=4.4700 83 a=3.7200 72 a=5.0000 83 a=3.7200 70 a=4.4700 79 a=5.4200	(gubergren)
Mx 	138.2259
My 	299.9564
Set font 	Libertinus Serif;10;400;Italic;normal;;;LTR
T	51 74 72 73 85 w=22.1200	(Right)
Mx 	162.8459
Mx 	4.5700
Mx 	2.6400
Mx 	2.7100
Mx 	5.0000
Mx 	5.4200
Mx 	4.5400
Mx 	5.0600
T	66 a=4.5700 77 a=2.6400 74 a=2.7100 72 a=5.0000 79 a=5.4200 70 a=4.4700 69 a=5.0600	(aligned)
Mx 	195.2859
Set font 	Libertinus Serif;10;400;;normal;;;LTR
Mx 	2.6400
Mx 	5.0400
Mx 	3.6400
Mx 	4.4700
Mx 	7.9000
T	77 a=2.6400 80 a=5.0400 83 a=3.7200 70 a=4.4700 78 a=7.9000	(lorem)
Mx 	221.4759
T	74 81 84 86 78 w=25.0100	(ipsum)
Mx 	248.9859
T	69 80 77 80 83 w=21.5000	(dolor)
Mx 	272.9859
T	84 74 85 w=9.7700	(sit)
End page
Finish
